From 04f6a1aa23e22f1271f7fac98d2e53a9caba26f6 Mon Sep 17 00:00:00 2001
From: Junliang Yan <junyan@redhat.com>
Date: Wed, 23 Feb 2022 10:41:54 -0500
Subject: [PATCH] s390x: [baseline] Add baseline test for s390x

Change-Id: Iebe587955aad8445cd22598a3e2930ca9444e792
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3484702
Reviewed-by: Victor Gomes <victorgomes@chromium.org>
Commit-Queue: Junliang Yan <junyan@redhat.com>
Cr-Commit-Position: refs/heads/main@{#79269}
---
 test/mjsunit/mjsunit.status | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mjsunit/mjsunit.status b/test/mjsunit/mjsunit.status
index f372e01478..de2a0e7c7c 100644
--- a/test/mjsunit/mjsunit.status
+++ b/test/mjsunit/mjsunit.status
@@ -1545,7 +1545,7 @@
 
 ##############################################################################
 # TODO(v8:11421): Port baseline compiler to other architectures.
-['arch not in (x64, arm64, ia32, arm, mips64el, mipsel, riscv64, loong64)', {
+['arch not in (x64, arm64, ia32, arm, mips64el, mipsel, riscv64, loong64, s390x)', {
   'baseline/*': [SKIP],
   'regress/regress-1242306': [SKIP],
 }],
-- 
2.35.1

